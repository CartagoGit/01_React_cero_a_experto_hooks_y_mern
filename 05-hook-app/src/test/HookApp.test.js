
import React from "react";
import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe("Pruebas en <HookApp />", () => {
    let wrapper;

    const fillWrapper = () => {
        wrapper = shallow(<HookApp />);
    };
    beforeEach(() => fillWrapper());

    test("Deberia cargar el snapshoot", () => {
        expect(wrapper).toMatchSnapshot();
    });

});
