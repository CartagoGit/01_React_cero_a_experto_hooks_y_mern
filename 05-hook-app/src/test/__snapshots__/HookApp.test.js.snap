// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> Deberia cargar el snapshoot 1`] = `
<div>
  <h1>
    Hola Mundo!
  </h1>
</div>
`;
